tasks/CellRegMap.wdl

```diff
@@ -0,0 +1,113 @@
+version development
+
+'''
+Tasks to intersect all input files and perform CellRegMap 
+functions for a specific chromosome-gene pair
+'''
+
+task RunInteraction { # CellRegMap's run_interaction()
+    input {
+        String chrom 
+        String geneName
+        File sampleMappingFile
+        File genotypeFile
+        File genotypeFilesBim
+        File genotypeFilesFam
+        File phenotypeFile
+        File contextFile
+        File kinshipFile
+        File featureVariantFile 
+        Int nContexts = 10 
+    }
+
+    command <<<
+        # make sure secondaries are paired together
+        cp -f '~{genotypeFilesBim}' "$(echo '~{genotypeFile}' | sed 's/\.[^.]*$//').bim"
+        cp -f '~{genotypeFilesFam}' "$(echo '~{genotypeFile}' | sed 's/\.[^.]*$//').fam"
+
+        # for now, use conda, but when we're closer,
+        # remove this in favor of 'container' in the runtime section
+        eval "$(conda shell.bash hook)" 
+        conda activate cellregmap_notebook 
+
+        python /share/ScratchGeneral/anncuo/github_repos/CellRegMap_pipeline/images/cellregmap/run_interaction.py \
+            --chrom ~{chrom} \
+            --gene-name ~{geneName} \
+            --sample-mapping-file ~{sampleMappingFile} \
+            --genotype-file ~{genotypeFile} \
+            --phenotype-file ~{phenotypeFile} \
+            --context-file ~{contextFile} \
+            --kinship-file ~{kinshipFile} \
+            --feature-variant-file ~{featureVariantFile} \
+            --n-contexts ~{nContexts} 
+    >>>
+
+    output {
+        File geneOutput = geneName + ".csv"
+    }
+
+    runtime {
+        # container: "annacuomo/limix:dev"
+
+        # static
+        memory: "400Gb"
+        
+        # # calculated
```

Comment:
Are you able to calculate the memory required for this task? Or can we clean up the comment here?